proxy/proxy.go

```diff
@@ -946,12 +944,11 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 		ctx.proxySpan.LogKV("event", "error", "message", err.Error())
 
 		if perr, ok := err.(*proxyError); ok {
-			p.log.Errorf("Failed to do backend roundtrip to %s: %v", ctx.route.Backend, perr)
 			//p.lb.AddHealthcheck(ctx.route.Backend)
+			perr.err = fmt.Errorf("failed to do backend roundtrip to %s: %w", ctx.route.Backend, perr.err)
```

Comment:
Here I think we better use `req.URL` or something because `ctx.route.Backend` is only applicable to the network backend - i.e. it is empty for loadbalanced backend

Reply:
yeah seems that this is a leftover from the old days when we had no loadbalanced backend

Reply:
I would go for req.URL.Host, because we don't need the specific URL and we would log basic auth credentials etc.